packages/web-components/fast-components/src/tree-item/tree-item.styles.ts

```diff
@@ -116,13 +130,15 @@ export const TreeItemStyles = css`
         ${
             /* Width and Height should be based off calc(glyph-size-number + (design-unit * 4) * 1px) - 
             update when density story is figured out */ ""
-        } width: var(--expand-collapse-button-size);
-        height: var(--expand-collapse-button-size);
+        } width: 28px;
+        height: 28px;
```

Comment:
hmm that should be changed at the definition of the css var, will fix, the size of the button needs to be reduced though to match the design of this hover state change.

Reply:
The change in size at all, or the change from var to constant? If the former, it's because previously the button didn't have its own backplate, but now that it does it doesn't visually fit correctly in the available space.

Reply:
Sorry it was using the expand-collapse bit, the current version anyway, but it needs to be less than heightNumber, fixing